software/rfcs/rocketCtl.md

```diff
@@ -0,0 +1,68 @@
+# RocketCtl
+
+RocketCtl or RocketControl is our system for configuring Apex systems before launch. It can control much of the setup of the avionics without needing us to recompile the code. 
+
+## Pre-reading
+
+1. Knowing that UART = Serial comminication = sending data on a wire to the avionics.
+
+2. What an RPC is.
+
+3. [Checksums](https://www.geeksforgeeks.org/error-detection-code-checksum/)
+
+## Why
+
+We want to get the Apex flight recorder certified as an official altimeter by UKRA, so we don't need to use 3rd party altimeters. This means that the firmware (including any on board config files) mustn't change between flights. But what if we want to change some minor details?
+
+## Main Proposal
+
+Ok so we can't change anything on the board between flights now, so what if the config wasn't stored on the board? What I'm proposing is that we store the config in a file somewhere on Github, and before a flight we can tell Apex that it needs to be configured in a specific way via serial communication.
```

Comment:
commericial altimeters have features similar to this so UKRA shouldn't beef us

AltusMetrum use a web server based approach I believe - you can connect to the altimeter over wifi and control params that way

Reply:
Aight well we just bring that up if/when they do beef us